src/packages/cli/src/CLI.ts

```diff
@@ -136,55 +144,59 @@ export class CLI implements Command {
       Browse your data
       ${chalk.dim('$')} prisma studio
 
+      Create and apply a migration for your database
+      ${chalk.dim('$')} prisma migrate dev --early-access-feature
+  
       Push the Prisma schema state to the database
       ${chalk.dim('$')} prisma db push --preview-feature
   `)
 
-  private static experimentalHelp = format(`
-    ${process.platform === 'win32' ? '' : chalk.bold.green('◭  ')
-    }Prisma is a modern DB toolkit to query, migrate and model your database (${link(
-      'https://prisma.io',
-    )})
+  // private static experimentalHelp = format(`
```

Comment:
Let's cleanup

Reply:
Will cleanup this when we get closer to 2.13